README.md

```diff
@@ -43,7 +43,8 @@ If you are not in your virtual env, the global Python environment will be used w
 your local package imports in with the 3rd party package imports.
 
 ```console
-usage: pyfmt [-h] [--check] [--line-length LINE_LENGTH]
+usage: pyfmt [-h] [--skip] [--isort] [--black] [--check] [--line-length LINE_LENGTH]
```

Comment:
let's get the kwargs on their own line, alphabetized.

args can still live on that top line, but alphabetize